module/VuFind/src/VuFind/Db/Service/RatingsServiceInterface.php

```diff
@@ -101,4 +111,21 @@ public function addOrUpdateRating(
         UserEntityInterface|int $userOrId,
         ?int $rating
     ): int;
+
+    /**
+     * Get a paginated result of all ratings by user id
```

Comment:
Add punctuation for consistency.

```suggestion
     * Get a paginated result of all ratings by user id.
```